src/docs/reference/modules/Recipes/README.md

```diff
@@ -353,6 +359,28 @@ The CustomSettings step allows you to populate your custom settings with initial
     }
 ```
 
+### Recipes Step
+
+The Recipes step allows you to execute other recipes from the current recipe. You can use this to modularize your recipes. E.g. instead of having a single large setup recipe you can put content into multiple smaller ones and execute them from the setup recipe.
+
+```json
+    {
+      "name": "recipes",
+      "Values": [
+        {
+          "executionid": "MyApp",
```

Comment:
Would it make sense to use
```suggestion
          "executionid": "[js: uuid()]",
```
here?

Reply:
Not even sure you need to specify that. Can you try without?

Reply:
You need to have it, yes, because `RecipesStep`'s code needs it when operating on the `recipesDictionary` dictionary.